R/tm_variable_browser.R

```diff
@@ -1,37 +1,28 @@
-#' Variable Browser Teal Module
+#' Variable browser `teal` module
 #'
-#' The variable browser provides a table with variable names and labels and a
-#' plot that visualizes the content of a particular variable.
-#' specifically designed for use with `data.frames`.
+#' Module provides provides a detailed summary and visualization of variable distributions
+#' for `data.frame` objects, with interactive features to customize analysis.
 #'
-#' @details Numeric columns with fewer than 30 distinct values can be treated as either factors
-#' or numbers with a checkbox allowing users to switch how they are treated (if < 6 unique values
-#' then the default is categorical, otherwise it is numeric).
+#' Numeric columns with fewer than 30 distinct values can be treated as either discrete
+#' or continuous with a checkbox allowing users to switch how they are treated(if < 6 unique values
+#' then the default is discrete, otherwise it is continuous).
```

Comment:
Description is still unclear for me. I opened example app and I don't see the difference between numerics with <6 unique values (e.g. mtcars$cyl) and <30 (mtcars$hp).